action.yml

```diff
@@ -127,7 +127,7 @@ runs:
 
     - name: Install nix
       if: inputs.skip-nix-installation == 'false'
-      uses: DeterminateSystems/nix-installer-action@cd46bde16ab981b0a7b2dce0574509104543276e  # v9
+      uses: DeterminateSystems/nix-installer-action@v10  # v9
```

Comment:
Can we pin this to a sha? See https://github.com/jetify-com/devbox-install-action/pull/32